packages/strapi-design-system/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -11,19 +11,23 @@ const CrumbWrapper = styled(Flex)`
   svg {
     height: 10px;
     width: 10px;
-  }
-  svg path {
-    fill: ${({ theme }) => theme.colors.neutral300};
+    path {
+      fill: ${({ theme }) => theme.colors.neutral500};
+    }
   }
   :last-of-type ${Box} {
     display: none;
   }
+  :last-of-type ${Typography} {
+    color: ${({ theme }) => theme.colors.neutral800};
+    font-weight: ${({ theme }) => theme.fontWeights.bold};
+  }
```

Comment:
mmh not sure I understand "`we are moving knowledge about the actual Crumb into the parent level`"
this is the v1 component and this styling is done on `Crumb` component, not on `Breadcrumbs`.
this pattern was already there before my changes when removing the last icon here:
<img width="192" alt="image" src="https://user-images.githubusercontent.com/71838159/178686011-79faf1cb-46d8-4207-bb39-abffa596c1fe.png">

about updating this logic by passing a prop to `Crumb`, why not but since it would change the API (if you don't pass isLast then you would have the last icon appearing) I don't think we should do it for Breadcrumbs v1 component.
I could implement it for the v2 one though, wdyt?

Reply:
Ah, my bad then. Yeah, for the v2 version that would be great :)